builder/common/run_config.go

```diff
@@ -390,11 +411,11 @@ type RunConfig struct {
 	// subnet-12345def, where Packer will launch the EC2 instance. This field is
 	// required if you are using an non-default VPC.
 	SubnetId string `mapstructure:"subnet_id" required:"false"`
-	// [Tenancy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/dedicated-instance.html) used
-	// when Packer launches the EC2 instance, allowing it to be launched on dedicated hardware.
-	//
-	// The default is "default", meaning shared tenancy. Allowed values are "default",
-	// "dedicated" and "host".
+	// [LicenseSpecification](#license-specification)
+	LicenseSpecifications []LicenseSpecification `mapstructure:"license_specifications" required:"false"`
+	// [Placement](#placement)
+	Placement Placement `mapstructure:"placement" required:"false"`
```

Comment:
I think it could be nice to add a little code snippet here to show how to use it.